mass_mailing_partner/models/mailing_contact.py

```diff
@@ -34,47 +34,75 @@ def _check_partner_id_list_ids(self):
                     )
 
     @api.onchange("partner_id")
-    def _onchange_partner_mass_mailing_partner(self):
+    def _onchange_partner_mass_mailing_partner(self, change_tag_bool=True):
```

Comment:
Is this really necessary?
In any case, I think the right thing to do would be to use context to do this.

Reply:
updated